build/builders.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using Docker Build
+description: About building with existing builders
+keywords: build, buildx, buildkit
+---
+
+
+## Docker Build Components
+
+Currently Engine holds two build components:
+* Legacy Builder - Old builder implementation. It's executed through the `docker build` command.
+* Buildkit - Active by default in new and latest installations of Docker. It's executed through the `docker buildx build` command.
+
+### Docker Build Default Configuration
+
+The backend you have active depends on your Docker's installation.
+If you have installed:
+
+* __Docker Desktop (all platforms)__: BuildKit is enabled by default; no additional configuration is required.
+* In Linux, if you have:
+  * Engine 18.09 and higher you have BuildKit enabled by default.
+  * Engine releases older than 18.09 only include the Legacy Builder component, so we highly recommend you upgrade your installation or alternatively install Docker Desktop.
+
+
+### Docker Build Configuration
+
+Backend configuration is performed using the `DOCKER_BUILDKIT` environment variable. 
+
+If not yet set as default, to set the BuildKit as the default backend behind the `docker build` command, you can:
+
+__Method 1__
+
+From your console, run:
+
+```console
+$ DOCKER_BUILDKIT=1 docker build .
+```
+` 
+Depending on the value for this variable:
+* __DOCKER_BUILDKIT=1__ - BuildKit is executed both when you issue the `docker build` and `docker buildx build` command. With this configuration `docker build` becomes an alias for `docker buildx build`.
+* __DOCKER_BUILDKIT=0__ - When you issue `docker build` the build is executed using the Legacy Builder and BuildKit when you issue the `docker buildx build` command.
+
+> See this [deprecation notice](https://github.com/docker/cli/blob/master/docs/deprecated.md#legacy-builder-for-linux-images).
```

Comment:
We should not link to external markdown files but internal link of our doc.

Reply:
I'm not sure what this is anymore, was this your earlier content?